### ISSUE TITLE
Choroidal neovascular membrane

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7510]
**Preferred term label:**
Choroidal neovascularization

**Synonyms**
Choroidal neovascular membrane

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added synonym
